ui/widgets/nuxeo-selectivity.js

```diff
@@ -7214,16 +7214,27 @@ typedArrayTags[weakMapTag] = false;
 
       // scope styles
       this.scopeSubtree(this.$.input, true);
+
+      // attach event listener to scroller parent to hide dropdown
+      this._scrollParent = this._getScrollParent(this);
```

Comment:
I guess we don't need the to pass `this` as a parameter here?

Reply:
Nope, you're right. I'll remove it.